data_enrichment.py

```diff
@@ -185,7 +194,12 @@ def format_dirigeants_pp(list_dirigeants_pp_sqlite, list_all_dirigeants=[]):
         else:
             logging.info(f'Missing dirigeants names for ****** {dirigeant_pp["siren"]}')
 
-    dirigeants_pp_processed = drop_duplicates(dirigeants_pp_processed)
+    if dirigeants_pp_processed:
+        # Drop exact duplicates
```

Comment:
Either rename function or add detailled comments.

`# Case when two dirigeant have exactly the same fields/value`

`# Case when two dirigeant have partially the same fields/value (eg. same name, and fistname, different date or qualities)`